supernode/config/constants.go

```diff
@@ -136,3 +136,9 @@ const (
 	// unit: MB/s
 	DefaultMaxBandwidth = 200 * rate.MB
 )
+
+// Pattern
+const (
+	P2pPattern = int32(0)
```

Comment:
make it to new type `Pattern` is better to understand

Reply:
DefaultPattern？ or P2pPattern = int32(1) not int32(0)

Reply:
type Pattern int32
const (
	P2pPattern = Pattern(0)
)
func x(p Pattern) {
}